### ISSUE TITLE
Cocoapods support?

### ISSUE THREAD

**Issue by hernangonzalez** [SwiftOnEdge/Reflex#1]
Hi,

Are you planning on making it available on Cocoapods?

Thanks,
H.

**Comment by cloutiertyler** [comment]
This is a good question. I would definitely consider it. Currently I'm waiting to see how the Swift Package Manager evolves, but I'm certainly not opposed to supporting Cocoapods. Reflex actually already supports Carthage, although I haven't advertised that in the Readme. I'll have to update that.

Pull requests are welcome too!

**Comment by hernangonzalez** [comment]
Hi @TheArtOfEngineering,  good to know. I'll try to work on a pod PR then. 

BTW, thanks for putting this out, I've been using RxSwift though its size & overhead makes me uneasy. I've actually reduce its usage on my project because of this (keeping it out of UI code).  

Feel free to close this. I'll probably create a new one once I get it done.

H.

**Comment by cloutiertyler** [comment]
Yeah the idea here was to make a lighter weight reactive framework which could be used as a simple callback system if desired. That way you could sort of scale up your commitment to the reactive paradigm. Ideally it wouldn't be opinionated at all.

**Comment by cloutiertyler** [comment]
Thanks for the PR! I'm going to add a section to the readme on how to install by the different methods and in general add more and better features.
